common/src/options.ts

```diff
@@ -65,6 +66,7 @@ export const defaultOptions: Options = {
   sikeRetries: 0,
   promptSkipping: true,
   minPlayers: 3,
+  recPlayers: 5,
   maxPlayers: 12,
```

Comment:
This can probably be increased to like 20, as it's no longer acting as a shield to user enjoyment, and now just acts as more of a sanity check

Reply:
I assume you mean maxPlayers so I will do that